Indico/Entity/Submission.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Indico.Types;
+
+namespace Indico.Entity
+{
+    public class Submission
+    {
+        public int Id { get; set; }
+        public int DatasetId { get; set; }
+        public int WorkflowId { get; set; }
+        public SubmissionStatus Status { get; set; }
+        public string InputFile { get; set; }
+        public string InputFilename { get; set; }
+        public string ResultFile { get; set; }
+        public bool Retrieved { get; set; } = false;
```

Comment:
Is there an error if there isn't a default here?

Reply:
Yeah, but it can be made nullable with bool? instead to allow null values